### DISCUSSION TITLE
Minha rede social para artistas

### DISCUSSION THREAD

**Discussion by yuri014** [frontendbr/forum#2123] [Mostre seu trabalho]
Eu fiz uma rede social para ajudar artes independentes e ajudar pessoas a descobrir artes novas com facilidade. Ela se chama [Comp-Art](https://www.comp-art.art/).

Eu usei Next.js e Apollo no front-end e Apollo express e mongoose no back-end.

Eu fiz uma [thread](https://twitter.com/ahmanoseil1/status/1420401562665562117) falando melhor sobre algumas coisas que aprendi durante esse processo.

Caso queiram saber mais aqui tem um link explicando quase tudo: https://twitter.com/CompArt_/status/1421571802246139904.

**Comment by nazarepiedady** [comment]
É um boa ideia, vale a pena investir nela, é possível que nasça algo único e novo, mas antes tens que dar uns ajustes em seu visual e marca tipo:

* Usar um nome mais sugestivo como: **artland**, **beauty**, **beautists**, **beartist**, **ateliê**, **atelier**.
* Usar ilustrações menos abstratas, que possuam expressão (personagens com rostos, objetos com algum detalhe mínimo).
* Página Inicial, uma melhor estrutura de informação.
